pkg/storage/utils/decomposedfs/upload.go

```diff
@@ -185,21 +186,24 @@ func (fs *Decomposedfs) NewUpload(ctx context.Context, info tusd.FileInfo) (uplo
 	log := appctx.GetLogger(ctx)
 	log.Debug().Interface("info", info).Msg("Decomposedfs: NewUpload")
 
-	fn := info.MetaData["filename"]
-	if fn == "" {
+	// sanity checks
```

Comment:
spureous comment

Reply:
Removed in https://github.com/cs3org/reva/pull/2383